--- apidoc.js.orig      2023-08-16 06:03:55.000000000 -0400
+++ apidoc.js   2023-12-26 14:45:47.202566775 -0500
@@ -47579,6 +47579,37 @@
                            "type" : "string",
                            "typetext" : "<string>"
                         },
+                        "freenas_user" : {
+                           "description" : "FreeNAS user for API access",
+                           "optional" : 1,
+                           "type" : "string",
+                           "typetext" : "<string>"
+                        },
+                        "freenas_password" : {
+                           "description" : "FreeNAS password for API access (Deprecated)",
+                           "optional" : 1,
+                           "type" : "string",
+                           "typetext" : "<string>"
+                        },
+                        "truenas_secret" : {
+                           "description" : "TrueNAS Secret for API access",
+                           "optional" : 1,
+                           "type" : "string",
+                           "typetext" : "<string>"
+                        },
+                        "freenas_use_ssl" : {
+                           "description" : "FreeNAS API access via SSL",
+                           "optional" : 1,
+                           "type" : "boolean",
+                           "typetext" : "<boolean>"
+                        },
+                        "freenas_apiv4_host" : {
+                           "description" : "FreeNAS API Host via IPv4",
+                           "format" : "address",
+                           "optional" : 1,
+                           "type" : "string",
+                           "typetext" : "<string>"
+                        },
                         "fuse" : {
                            "description" : "Mount CephFS through FUSE.",
                            "optional" : 1,
@@ -47798,6 +47829,12 @@
                            "type" : "boolean",
                            "typetext" : "<boolean>"
                         },
+                        "target" : {
+                           "description" : "iSCSI target.",
+                           "optional" : 1,
+                           "type" : "string",
+                           "typetext" : "<string>"
+                        },
                         "transport" : {
                            "description" : "Gluster transport: tcp or rdma",
                            "enum" : [
@@ -48097,6 +48134,37 @@
                      "optional" : 1,
                      "type" : "string",
                      "typetext" : "<string>"
+                  },
+                  "freenas_user" : {
+                     "description" : "FreeNAS user for API access",
+                     "optional" : 1,
+                     "type" : "string",
+                     "typetext" : "<string>"
+                  },
+                  "freenas_password" : {
+                     "description" : "FreeNAS password for API access (Deprecated)",
+                     "optional" : 1,
+                     "type" : "string",
+                     "typetext" : "<string>"
+                  },
+                  "truenas_secret" : {
+                     "description" : "TrueNAS secret for API access",
+                     "optional" : 1,
+                     "type" : "string",
+                     "typetext" : "<string>"
+                  },
+                  "freenas_use_ssl" : {
+                     "description" : "FreeNAS API access via SSL",
+                     "optional" : 1,
+                     "type" : "boolean",
+                     "typetext" : "<boolean>"
+                  },
+                  "freenas_apiv4_host" : {
+                     "description" : "FreeNAS API Host via IPv4",
+                     "format" : "address",
+                     "optional" : 1,
+                     "type" : "string",
+                     "typetext" : "<string>"
                   },
                   "fuse" : {
                      "description" : "Mount CephFS through FUSE.",
